e2e/internal/client/cli.go

```diff
@@ -284,10 +285,15 @@ copilot env init
 	--prod (optionally)
 */
 func (cli *CLI) EnvInit(opts *EnvInitRequest) (string, error) {
+	useDefault := "true"
+	if opts.CustomizedEnv {
+		useDefault = "false"
+	}
 	commands := []string{"env", "init",
 		"--name", opts.EnvName,
 		"--app", opts.AppName,
 		"--profile", opts.Profile,
+		"--no-custom-resources", useDefault,
```

Comment:
nit: 
```suggestion
		"--no-custom-resources", !opts.CustomizedEnv,
```

Reply:
I know this is out of scope but do we need `--no-custom-resources` to take an argument at all here? Couldn't it just be a boolean flag, and not require customers to type "true" or "false"? Or am I misunderstanding how the e2es work?

Reply:
it doesn't need an argument... it's just by taking an argument can make it easier in e2e test

Reply:
addressed